coralproject/talk
# [CORL-2886] More generic title for "Log In" popout

## What does this PR do?
This PR changes the page title (as in the `<title></title>` element in the HTML head) from "Coral" to "Log In" to better suite users who want Coral to appear to be part of their own site. It also allows for custom variables to be passed into our server side templates.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
Yes. There is now an environment variable `SIGNIN_WINDOW_TITLE` that defaults to "Sign in", which should allow for overriding this value for localization/branding purposes.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
As a commenter, click the "Sign in to join conversation" button, and note that the title of the new window that opens is "Log In" rather than "Coral"
 
## How do we deploy this PR?
No special considerations should be needed.
